### DISCUSSION TITLE
Copy Lines replaces a string literal with "null"

### DISCUSSION THREAD

**Discussion by i-ky** [community/community#55864] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

The bug can be observed [here](https://github.com/zulip/python-zulip-api/blob/02586f1d348dbcab3bc8d30b805fffd48a524e18/zulip/integrations/rss/rss-bot#L173-L178). If one clicks _Copy Lines_, then a following code will be copied:
```python
    content = null.format(
        entry.title,
        entry.link,
        strip_tags(body),
        entry.link,
    )  # type: str
```
The code is actually:
```python
    content = "**[{}]({})**\n{}\n{}".format(
        entry.title,
        entry.link,
        strip_tags(body),
        entry.link,
    )  # type: str
```
Note how `"**[{}]({})**\n{}\n{}"` got replaced with `null`.

**Comment by AdamShwert** [comment, marked as answer]
We have a resolution for this bug ready to roll at at some point soon. Thank you for the report!

**Reply by i-ky** [reply]
Seems to work fine now, thanks!
